pkg/cdi/default-cache_test.go

```diff
@@ -115,7 +115,8 @@ devices:
 					filepath.Join(dir, "run"),
 				),
 			}
-			Configure(opts...)
+			err = Configure(opts...)
+			require.Nil(t, err, "unexpected Configure() error")
```

Comment:
`require.NoError` would be more explicit here in my opinion.

Reply:
Thanks for spotting it. Updated accordingly.